tests/flask_integration/flask.py

```diff
@@ -125,9 +123,9 @@ def login_member(
             password = "password123"
         if email is None:
             email = self.email_generator.get_random_email()
-        if member is None:
-            member = self.member_generator.create_member(
-                password=password, email=email, confirmed=False
+        if not self.database_gateway.get_members().with_email_address(email).first():
```

Comment:
just a suggestion to avoid having two db calls (here and l. 144): wouldn't it be possible to have a return var (sth like member _id) and then do sth like
`member = self.database_gateway.get_members().with_email_address(email).first()`
`if not member {`
`    member_id = self.member_generator.create_member`
`} else {`
`    member_id = member.id`
`}`
and then later just return member_id? just from looking at the code, did not test if that works though.

Reply:
You are right. The second db call was still from the times when we did not simply return a UUID, but a record object, that had to be recreated from the db before returning it.